src/Module/Admin/Site.php

```diff
@@ -467,7 +468,10 @@ protected function content(array $request = []): string
 			'$show_help'        => ['show_help', DI::l10n()->t('Show help entry from navigation menu'), !DI::config()->get('system', 'hide_help'), DI::l10n()->t('Displays the menu entry for the Help pages from the navigation menu. It is always accessible by calling /help directly.')],
 			'$singleuser'       => ['singleuser', DI::l10n()->t('Single user instance'), DI::config()->get('system', 'singleuser', '---'), DI::l10n()->t('Make this instance multi-user or single-user for the named user'), $user_names],
 
-			'$maximagesize'     => ['maximagesize', DI::l10n()->t('Maximum image size'), DI::config()->get('system', 'maximagesize'), DI::l10n()->t('Maximum size in bytes of uploaded images. Default is 0, which means no limits. Be aware that this setting does not affect server-side upload limits.')],
+			'$maximagesize'     => ['maximagesize', DI::l10n()->t('Maximum image size'), DI::config()->get('system', 'maximagesize'), DI::l10n()->t('Maximum size in bytes of uploaded images. Default is 0, which means no limits.') .
+													DI::l10n()->t('The value of <code>upload_max_filesize</code> in your <code>PHP.ini</code> needs be set to at least the desired limit. ') .
+													DI::l10n()->t('Currently <code>upload_max_filesize</code> is set to ' . Strings::getBytesFromShorthand(ini_get('upload_max_filesize')) . ' (' . ini_get('upload_max_filesize') . 'B). '),
+													'1 kB = 1 KiB = 1024 Bytes'],
```

Comment:
Where does this get displayed?

Reply:
On the "usage info", the red star at the end of the input field title. This was already available in the templates, so I just used it.

Reply:
![Inkedfriendica_usage_message_maxfilesize](https://user-images.githubusercontent.com/117765478/204168011-4e71d510-498c-4469-b05e-c1f8e14fbc0d.jpg)

Reply:
I see. This is meant as a required field, not sure how visible it is. Additionally, the field itself doesn't use shorthands yet, so this may prompt incorrect input in the field.

Reply:
Is there a field that can deal shorthand? As mentioned, I would really appreciate to use shorthand notation or drop down menu for the desired prefix.
In any way, I am out for the next hours, it's late in my TZ :-) 
Will be back some day on monday.

Reply:
There's no field that can deal with shorthands at the moment, but now that we have the `Strings::getBytesFromShorthand` method, we can use it to convert config values including this one when we actually use it.

Reply:
Okay, thank you. As I see it, two more things to do:

1. Add handling for shorthand: It would be straight forward to do by adding the new `Strings::getBytesFromShorthand())` to `Site.php:71` but we only can do it when it is guaranteed that the entered value is in correct format. And this is not guaranteed at all. **Where do we therefore best place a regex check, e.g. `\d+([g,m,k]|[G,M,L]){1}\s` for the entered value and reject the input if the check fails?** , some kind of frontend js check? I am not familiar with js.
2. Iff 1. is implemented, then it makes sense to put the prefix definition of `1kB = 1KiB = 1024 B` directly beneath the input field. And of course, the info that shorthand can be used.  :-)